docs/creating-new/autotools.rst

```diff
@@ -0,0 +1,99 @@
+Autotools
+=========
+
+Very often, you will come across a package that uses autotools as its build
+system and does not support CMake builds. Although Hunter prefers CMake builds
+when possible, it does support autotools projects when no CMake build is
+available. Here is how to do it.
+
+.. code-block:: cmake
+
+    # !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+    include(hunter_add_version)
+    include(hunter_configuration_types)
+    include(hunter_pick_scheme)
+    include(hunter_download)
+    include(hunter_cacheable)
+    include(hunter_cmake_args)
+
+    hunter_add_version(
+        PACKAGE_NAME
+        foo
+        VERSION
+        "1.2.3"
+        URL
+        "https://example.com/foo-1.2.3.tar.gz"
+        SHA1
+        da39a3ee5e6b4b0d3255bfef95601890afd80709
+    )
+
+    # More versions...
+
+    # Optional platform customization.
+    if (ANDROID OR IOS)
+      hunter_cmake_args(
+          foo
+          CMAKE_ARGS
+              EXTRA_FLAGS=--enable-x
+      )
+    endif()
+
+    hunter_configuration_types(foo CONFIGURATION_TYPES Release)
+    hunter_pick_scheme(DEFAULT url_sha1_autotools)
+    hunter_cacheable(foo)
+    hunter_download(PACKAGE_NAME foo)
+
+Note that the build may not be cacheable if autotools generation expands
+absolute paths. Try using ``hunter_cacheable`` and see if it works.
+
+**Important**: by default, this build will not provide any CMake config.
+Therefore, projects using autotools **must provide a file ``Findfoo.cmake`` in
+the ``cmake/find`` directory**. Dependent projects will use code similar to the
+following:
+
+.. code-block:: cmake
+
+    hunter_add_package(foo)
+    find_package(foo REQUIRED)
+    add_executable(bar ${BAR_SOURCES})
+    target_include_directories(bar PUBLIC ${FOO_INCLUDE_DIRS})
+    target_link_libraries(bar PUBLIC ${FOO_LIBRARIES})
+
+However, many autotools projects generate pkg-config files. These can be used
+to generate a CMake config. For example, consider using the following in your
+package's ``hunter.cmake`` file:
+
+.. code-block:: cmake
+
+    hunter_cmake_args(
+        foo
+        CMAKE_ARGS
+            PKGCONFIG_EXPORT_TARGETS=foo
+    )
+
+In the above example, package ``foo`` generates a file ``foo.pc`` in the
+autotools build. CMake then uses ``foo.pc`` to generate a CMake config file
```

Comment:
> CMake then uses ``foo.pc`` to generate a CMake config file

It's not CMake, it's Hunter:
* Template https://github.com/ruslo/hunter/blob/0dc25c663ea30f597487912e4ed4c68eb78c7385/scripts/pkgconfig-export-targets.cmake.in#L8-L10
* Autotools build scheme: https://github.com/ruslo/hunter/blob/0dc25c663ea30f597487912e4ed4c68eb78c7385/cmake/schemes/url_sha1_autotools.cmake.in#L62-L70
* Helper module: https://github.com/ruslo/hunter/blob/0dc25c663ea30f597487912e4ed4c68eb78c7385/cmake/modules/hunter_pkgconfig_export_target.cmake#L15-L16